### DISCUSSION TITLE
[docs] Do we need a "Themes" page?

### DISCUSSION THREAD

**Discussion by methomps** [RedHat-UX/red-hat-design-system#1178] [General]
A page that discusses our various themes, like Light and Dark, and how to use them from a design and development perspective.

**Comment by methomps** [comment]
BLOCKED: Documentation needs written

**Comment by markcaron** [comment]
Converted to a discussion. Do we have a need for a general "theming" page?

**Reply by bennypowers** [reply]
Any docs we write on this must reference the [three colour themes](https://github.com/RedHat-UX/red-hat-design-system/blob/517119629bc7b96e9d4cb0360a7cc87ad3e20f4e/lib/context/color/consumer.ts#L16) and [nine (!) colour palettes](https://github.com/RedHat-UX/red-hat-design-system/blob/517119629bc7b96e9d4cb0360a7cc87ad3e20f4e/lib/context/color/provider.ts#L21) that we support, and it must clearly explain the difference between the two types
